content/en/platform/corda/5.0-beta/developing/getting-started/first-flow/first-flow.md

```diff
@@ -12,7 +12,7 @@ section_menu: corda-5-beta
 The `MyFirstFlow` and `MyFirstResponderFlow` flows are basic flows that illustrate the main features of Corda 5 flows.
 Many of the features will be familiar to those Developers who have written Corda 4 CorDapps. However, there are some important differences when using Corda 5:
 * Services are injected on an as-needed basis.
-* Flows are started via HTTP RPC, rather than a Java client sending Java classes over AMQP.
+* Flows are started via REST, rather than a Java client sending Java classes over AMQP.
```

Comment:
Would "Flows are started via a REST endpoint" be better? I'll leave that for you to decide though